spk/shairport/src/installer.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+
+# Package
+PACKAGE="shairport"
+DNAME="ShairPort"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+SSS="/var/packages/${PACKAGE}/scripts/start-stop-status"
+PATH="${INSTALL_DIR}/bin:/usr/local/bin:/bin:/usr/bin:/usr/syno/bin:${PATH}"
+TMP_DIR="${SYNOPKG_PKGDEST}/../../@tmp"
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+
+	# TODO maybe add an spk/shairport/src/alsa.conf with the correct configuration, and place that file during installation.
```

Comment:
The general approach to add a 'standard' file would look like this: https://github.com/SynoCommunity/spksrc/blob/master/spk/tt-rss/Makefile#L42 (put your file in `/spk/shairport/src`), then you can simply replace the current file during the installer.